daemon/daemon.go

```diff
@@ -12,14 +12,14 @@ import (
 	"sync"
 	"time"
 
-	aa_log "github.com/aaronland/go-log/v2"	
-	"github.com/aaronland/go-http-server"
+	server "github.com/aaronland/go-http-server"
```

Comment:
The prefix is unnecessary as `go-http-server` already exports the namespace and it is not in conflict with anything else.

Reply:
My editor (VSCode) automatically makes this change - any suggestions on how to prevent?  May by unnecessary but I see no reason not to explicitly label the namespace.